recipes/spdlog/all/conanfile.py

```diff
@@ -45,7 +45,7 @@ def configure(self):
             raise ConanInvalidConfiguration("wchar is not yet supported under windows")
 
     def requirements(self):
-        self.requires("fmt/6.0.0")
+            self.requires("fmt/6.0.0")
```

Comment:
```suggestion
        self.requires("fmt/6.0.0")
```

Reply:
bad indentation, you have added more 4 spaces due the previous `if` which was removed. Please, fix it, than we are good! Regards!

Reply:
Definitely, that happens if you once edit files in browser ...